chart/values.yaml

```diff
@@ -587,6 +587,34 @@ loki-stack:
               - __meta_kubernetes_pod_uid
               - __meta_kubernetes_pod_container_name
               target_label: __path__
+
+nats:
+  enabled: true
+  nats:
+    image:
+      repository: nats
+      pullPolicy: IfNotPresent
+    jetstream:
+      enabled: true
+      memStorage:
+        enabled: true
+        # Size of nats message is around 0.3 KB, so it can store around 10K messages.
+        size: "5Mi"
+      fileStorage:
+        enabled: false
+  cluster:
+    enabled: true
+    # Cluster name is required, by default will be release name.
+    name: "nats"
```

Comment:
can we leave it as default to use release name?

Reply:
removed